### PR TITLE
Gmail oauth flow shouldn't attempt to parse the command line.

### PR DESCRIPTION
**Pull request by cthulahoops**
If any command line arguments are passed to bugwarrior, gmail will try
to parse them and fail. It's not possible pass arguments to Oauth this
way anyway (because bugwarrior would reject the unrecognised command
line arguments), so we shouldn't try to do it. (I see no reason to pass
flags to OAuth here, if needed we'd have to find another way.)

Should resolve issue #488.

### CODE REVIEW

**Comment by pjf** [comment]
Tested!

- This resolves #488 for me, and now pushes me through the oauth-dance if I'm not authenticated, which is great!
- After running, this creates an appropriate credentials file inside my `~/.task` directory, but of course does not modify any issues. This matches our suggested behaviour on #488.

**tl;dr**: This works for me! Thank you so much!
